src/devices/Mcp3xxx/samples/Mcp3008.Sample.cs

```diff
@@ -21,7 +23,7 @@ static void Main(string[] args)
             using (SpiDevice spi = new SoftwareSpi(clk: 6, miso: 23, mosi: 5, cs: 24))
             // For hardware implementation replace it with following
             // using (SpiDevice spi = SpiDevice.Create(hardwareSpiSettings))
-            using (Mcp3008.Mcp3008 mcp = new Mcp3008.Mcp3008(spi))
+            using (Mcp3008 mcp = new Mcp3008(spi))
```

Comment:
that's why I asked to change the namespace so that we don't have to double type `Mcp3008` 😄 Thanks!